src/crypto-providers/ledgerTypes.ts

```diff
@@ -80,3 +69,44 @@ export type LedgerCertificate = {
   poolKeyHashHex?: string,
   poolRegistrationParams?: LedgerPoolParams,
 }
+
+export const enum LedgerCryptoProviderFeature {
+  BULK_EXPORT,
+  MULTI_ASSET,
+  OPTIONAL_TTL,
+  VALIDITY_INTERVAL_START,
+}
+
+export type LedgerStakingBlockchainPointer = {
+  blockIndex: number,
+  txIndex: number,
+  certificateIndex: number
+}
+
+export type LedgerToken = {
+  assetNameHex: string,
+  amountStr: string
+}
+
+export type LedgerAssetGroup = {
+  policyIdHex: string,
+  tokens: Array<LedgerToken>
+}
+
+export type LedgerTxOutputTypeAddress = {
+  amountStr: string,
+  tokenBundle: Array<LedgerAssetGroup>,
```

Comment:
I see other parts of the codebase have LedgerAssetGroup[] instead - I guess it's the same thing and I would stick to the more concise notation

Reply:
Refactored